Sources/StreamChatUI/Composer/WalletInputView/WalletStepper.swift

```diff
@@ -46,6 +46,7 @@ class WalletStepper: UIView {
 
     private var centerContainerXLayoutConstraint: NSLayoutConstraint?
     private var centerContainerYLayoutConstraint: NSLayoutConstraint?
+    private var decimalSeparator = Locale.current.decimalSeparator ?? "."
```

Comment:
@parth-simformsolutions  Please initialize decimalSeparator in single place and use it everywhere.